src/sentry/api/serializers/models/group.py

```diff
@@ -142,7 +142,7 @@ def get_attrs(self, item_list, user):
 
         result = {}
         for item in item_list:
-            active_date = item.active_at or item.last_seen
+            active_date = item.active_at or item.first_seen
```

Comment:
this seemingly was broken, according to my super old docstring on what active_at is supposed to mean